README.md

```diff
@@ -0,0 +1,41 @@
+### Installation
+
+First add the dependency to your `Cargo.toml`
+```toml
+[dependencies]
+...
+ffmpeg = "0.3.0"
+```
+Then update (will likely [take a while](https://stackoverflow.com/questions/53361052/how-do-i-debug-cargo-build-hanging-at-updating-crates-io-index) due to size of downloaded ffmpeg lib)
+```bash
+cargo update
+```
+
+### Usage
+
+```rust
+use std::{fs::File};
+
+fn main() {
+    ffmpeg::init().unwrap();
+    let file = File::open("path/to/your/video/file.mkv").unwrap();
+    match ffmpeg::format::io::input(file) {
```

Comment:
I think it should rather use `ffmpeg::format::input` instead of `ffmpeg::format::io::input`.